docs/quickstart.rst

```diff
@@ -20,42 +20,60 @@ Activate the `virtual environment <https://docs.python.org/3/tutorial/venv.html>
 
 	$ python -m pip install -e .
 
-Either installation method will install all of the dependencies which are required to run Scenic.
 If you will be developing Scenic, you will want to use a variant of this command: see :doc:`developing`.
 
+Either installation method will install all of the dependencies which are required to run Scenic, with the exception of `Blender <https://www.blender.org/>`_ and `OpenSCAD <https://openscad.org/>`_ which must also be installed on the system. This is best done by going to the download pages for `Blender <https://www.blender.org/download/>`__ and `OpenSCAD <https://openscad.org/downloads.html>`__.
+
 .. note::
 
-	If you are using Windows, or encounter any errors during installation or trying the examples below, please see our :doc:`install_notes` for suggestions.
+	If you are using Windows or an Apple silicon machine, or encounter any errors during installation or trying the examples below, please see our :doc:`install_notes` for suggestions.
 
 Trying Some Examples
 --------------------
 
 The Scenic repository contains many example scenarios, found in the :file:`examples` directory.
 They are organized by the simulator they are written for, e.g. :abbr:`GTA (Grand Theft Auto V)` or Webots; there are also cross-platform scenarios written for Scenic's abstract application domains, e.g. the :ref:`driving domain <driving_domain>`.
+
+
 Each simulator has a specialized Scenic interface which requires additional setup (see :ref:`simulators`); however, for convenience Scenic provides an easy way to visualize scenarios without running a simulator.
 Simply run :command:`scenic`, giving a path to a Scenic file:
 
 .. code-block:: console
 
-	$ scenic examples/gta/badlyParkedCar2.scenic
+	$ scenic examples/webots/vacuum/vacuum_simple.scenic
+
+This will compile the Scenic program and sample from it (which may take several seconds), displaying a schematic of the resulting scene. Since this a simple scenario designed to evaluate the performance of a robot vacuum, you should get something like this:
+
+.. image:: images/vacuumSimple.png
+	:width: 50%
+	:figclass: align-center
+
+The green cylinder is the ego vacuum, surrounded by various pieces of furniture in a room. The viewer can be moved around using the mouse, and the axes in the center are provided to help clarify orientation. If you close the window, Scenic will sample another scene from the same scenario and display it.
+This will repeat until you kill the generator (:kbd:`Control-c` in Linux; right-clicking on the Dock icon and selecting Quit on OS X).
```

Comment:
As we discussed, please update this. On MacOS it seems `:kbd:`Command-q` is the right action. (Although that doesn't work for the old 2D viewer, unfortunately; we should look to see if it's easy to tell Matplotlib how to respond to Command-Q).